### PR TITLE
Amib1 ee xxx fix

### PR DESCRIPTION
**Pull request by amaanibn-nasar1-nhs**

### CODE REVIEW

File: .github/workflows/ci-pipeline.yaml [hunk 1]
```diff
@@ -31,6 +31,9 @@ jobs:
       - name: Check types
         run: yarn tsc --noEmit
 
+      - name: Build package
+        run: yarn build
```
Comment (regularfry) [inline .github/workflows/ci-pipeline.yaml:35]: Is this going to fail under all cases that the `yarn tsc --noEmit` will?

**Review by regularfry — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-pipeline.yaml (hunk 1 above)
Comment (amaanibn-nasar1-nhs) [reply .github/workflows/ci-pipeline.yaml:35]: no it's slightly different
yarn build builds both our cjs and esm side whereas yarn tsc just checks the types work

Comment (regularfry) [reply .github/workflows/ci-pipeline.yaml:35]: Yeah, but I guess the question is whether the build checks the types as well.

Comment (amaanibn-nasar1-nhs) [reply .github/workflows/ci-pipeline.yaml:35]: yes it will

Comment (amaanibn-nasar1-nhs) [reply .github/workflows/ci-pipeline.yaml:35]: which I guess makes the check types step superfluous as build will do more

Comment (regularfry) [reply .github/workflows/ci-pipeline.yaml:32]: That's exactly what I was thinking - checking the types is exactly what we want, but we don't want to do it twice unless the timings need work.

**Review by regularfry — APPROVED** [review]

**Pull request merged**